src/style.css

```diff
@@ -0,0 +1,57 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.search {
+  width: 100%;
+  color: rgba(182, 172, 172, 0.14);
+  border: 1px solid rgb(232, 230, 230);
+  border-radius: 4px;
+  box-shadow: 0 1px 8px #7f7f7f26;
+  font-family: Avenir, sans-serif;
+  box-sizing: border-box;
+}
+
+.searchBig {
+  height: 70px;
+  background: url(images/Search.svg) 25px center no-repeat;
```

Comment:
you need to insert this icon as ::before

Reply:
fix it
https://github.com/mate-academy/layout_search-bar-airbnb/pull/2953/files#r1114676080